packages.go

```diff
@@ -9,15 +9,106 @@ import (
 	"time"
 )
 
+type screeningPackage string
+
+// ScreeningPackage ...
+type ScreeningPackage interface {
+	Message() string
+	Code() string
+}
+
+var packages = make(map[ScreeningPackage]string)
+
+// Message returns Message string from message
+// map for particular Package
+func (a screeningPackage) Message() string {
```

Comment:
@chandradeepak maybe change this to `String()`. And you actually don't need the `Code()` since you will only ever use the `driver_pro` and rest.

Reply:
@chandradeepak actually, change this to `Code()` and get rid of `Message()`.

Reply:
@magicalbanana  i am currently using the code not message. Since code is SLUG info for package which we are passing to create a report.
so should i keep code and delete message

Reply:
@chandradeepak so it's called `SLUG`? If it is, just name the function `SLUG()`